pkg/scalers/openstack_swift_scaler.go

```diff
@@ -0,0 +1,374 @@
+package scalers
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"path"
+	"strconv"
+	"strings"
+
+	"github.com/kedacore/keda/v2/pkg/scalers/openstack"
+	kedautil "github.com/kedacore/keda/v2/pkg/util"
+	v2beta2 "k8s.io/api/autoscaling/v2beta2"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/metrics/pkg/apis/external_metrics"
+
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+const (
+	defaultOnlyFiles         = false
+	defaultObjectCount       = 2
+	defaultObjectLimit       = ""
+	defaultObjectPrefix      = ""
+	defaultObjectDelimiter   = ""
+	defaultHTTPClientTimeout = 30
+)
+
+type swiftMetadata struct {
+	swiftURL          string
+	containerName     string
+	objectCount       int
+	objectPrefix      string
+	objectDelimiter   string
+	objectLimit       string
+	httpClientTimeout int
```

Comment:
For handling the http, could you please take a look on this PR: https://github.com/kedacore/keda/pull/1251

there's `kedautil.CreateHTTPClient(timeout time.Duration)` function.

Reply:
Done by 07fc5e4.